pydantic/_internal/_decorators.py

```diff
@@ -521,6 +529,28 @@ def update_from_config(self, config_wrapper: ConfigWrapper) -> None:
             computed_field_dec.info._update_from_config(config_wrapper, name)
 
 
+def is_method_overridden(model_dc: type[Any]) -> tuple[bool, set[Any] | None]:
```

Comment:
Can you please make this a method on `DecoratorInfos` so we can use `self`? We can then call it `find_duplicates` and return just a set, which may be empty.